cli/tests/data/dcos_universe2.toml

```diff
@@ -0,0 +1,9 @@
+[core]
```

Comment:
@tamarrow Let's not just execute a subset of the tests against version-2 of universe. All of the cli tests should be against version-2.x of universe.